openapi/resource-server.yaml

```
@@ -1012,6 +1019,125 @@ components:
+            assetScale: 2
+          createdAt: '2022-03-12T23:20:50.52Z'
+          updatedAt: '2022-04-01T10:24:36.11Z'
+          metadata:
+            description: APlusVideo subscription
+            externalRef: 'customer: 847458475'
+        - id: 'https://ilp.rafiki.money/outgoing-payments/0cffa5a4-58fd-4cc8-8e01-7145c72bf07c'
+          walletAddress: 'https://ilp.rafiki.money/alice/'
+          failed: false
+          receiver: 'https://ilp.rafiki.money/shoeshop/2fe92c6f-ef0d-487c-8759-3784eae6bce9'
+          debitAmount:
+            value: '7126'
+            assetCode: USD
+            assetScale: 2
+          sentAmount:
+            value: '7026'
+            assetCode: USD
+            assetScale: 2
+          grantSpentDebitAmount:
+            value: '7126'
+            assetCode: USD
+            assetScale: 2
+          grantSpentReceiveAmount:
+            value: '7026'
+            assetCode: USD
+            assetScale: 2
+          createdAt: '2022-03-12T23:20:50.52Z'
+          updatedAt: '2022-04-01T10:24:36.11Z'
+          metadata:
+            description: Thank you for your purchase at ShoeShop!
+            externalRef: INV2022-8943756
+      properties:
+        id:
+          type: string
+          format: uri
+          description: The URL identifying the outgoing payment.
+          readOnly: true
+        walletAddress:
+          type: string
+          format: uri
+          description: The URL of the wallet address from which this payment is sent.
+          readOnly: true
+        quoteId:
+          type: string
+          format: uri
+          description: The URL of the quote defining this payment's amounts.
+          readOnly: true
+        failed:
+          type: boolean
+          description: Describes whether the payment failed to send its full amount.
+          default: false
+        receiver:
+          $ref: ./schemas.yaml#/components/schemas/receiver
+          description: The URL of the incoming payment that is being paid.
+        receiveAmount:
+          $ref: ./schemas.yaml#/components/schemas/amount
+          description: The total amount that should be received by the receiver when this outgoing payment has been paid.
+        debitAmount:
+          $ref: ./schemas.yaml#/components/schemas/amount
+          description: The total amount that should be deducted from the sender's account when this outgoing payment has been paid.
+        sentAmount:
+          $ref: ./schemas.yaml#/components/schemas/amount
+          description: The total amount that has been sent under this outgoing payment.
+        grantSpentDebitAmount:
+          $ref: './schemas.yaml#/components/schemas/amount'
+          description: The amount already deducted from the sender's account against the outgoing payment.
+        grantSpentReceiveAmount:
+          $ref: './schemas.yaml#/components/schemas/amount'
+          description: The total amount already received against the outgoing payment.
```

```suggestion
          description: The total amount already received by receivers against the current outgoing payment grant.
```
There's definitely better wording than this, but I just wanted to highlight that it can be multiple receivers (across multiple outgoing payments) under the same outgoing payment grant

---

That seems like a good thing to emphasize and yeah "received by receivers" sounds a little clunky... this doesn't remove the usage of `received` and `receivers` but maybe sounds a bit better and makes it clear.

> The total amount already received (by all receivers) against the current outgoing payment grant.

At least these kinda redundant forms are more separated? 

Instead of "received" we could maybe say something like "collected" or "gained", "deposited to " etc. but I feel like "received" is consistent with our word-choice elsewhere. I guess we could use "incoming payment" instead of receiver?

> The total amount already received by all incoming payments against the current outgoing payment grant.

I think the first way is clearer IMO.

---

changed to first way for now but feel free to revise

> The total amount already received (by all receivers) against the current outgoing payment grant.